cypress/e2e/flarumTest.cy.js

```diff
@@ -0,0 +1,53 @@
+describe('template spec', () => {
+
+  beforeEach(() => {
+    cy.loadTestDataFromJson();
+    cy.visit('/')
+  })
+
+  it('testCase1', () => {
+    const menuOptions = Cypress.env('menuOptions');
+    const username = Cypress.env('username');
+    const password = Cypress.env('password');
+
+    cy.get('.Header-logo').should('exist')
+
+    cy.clickLink('.item-logIn > .Button > .Button-label:contains("Log In")')
+
+    cy.loginViaUi(username, password)
+
+    cy.clickLink(`.Button-label > .username:contains('${username}')`)
+
+    const dropdownMenu = '.item-session > .ButtonGroup > .Dropdown-menu'
+
+    menuOptions.forEach((dropdownMenu) => {
+      cy.get(dropdownMenu).each(($element) => {
+        cy.wrap($element).should('exist');
+      })
```

Comment:
Please pay attention to the distinction between the existence and visibility of elements. The check for existence may pass even if an element is not visible, which contradicts the expected result in the task. Therefore, we may miss a bug.